### DISCUSSION TITLE
Add more libinput options, especially "scroll_button" and "scroll_factor"

### DISCUSSION THREAD

**Discussion by Schweber** [YaLTeR/niri#304] [Ideas]
I'd like to have more options to change the behaviour of `libinput`, as this is a huge QoL improvement for me. For me, this is way more important than animations, looks etc. (i'm not using them anyway).

**Comment by YaLTeR** [comment]
You could try adding them yourself if you want; they should be pretty straightforward to add similarly to other libinput options, e.g. https://github.com/YaLTeR/niri/commit/cc83ff008d7500f38bce4e75c2dbcdfc1621cac4

**Comment by chillinbythetree** [comment]
Hey. I added the scroll_button property. I'm currently using it with my trackball and it works great. It will be available in the next release but you can use it now if you compile it from git.

https://github.com/YaLTeR/niri/pull/744

Someone is working on implementing scroll_factor. You can follow the progress here:

https://github.com/YaLTeR/niri/pull/730
